### ISSUE TITLE
[FEATURE REQUEST - Dearrow] Setting for the default state 

### ISSUE THREAD

**Issue by EDM115** [Vendicated/Vencord#2783]
### Content

A setting could be added to either enable dearrow on youtube embeds by default (current behavior) or disable it by default (and clicking on the button per embed)

### Request Agreement

- [X] I have read the requirements for opening an issue above

**Comment by surgedevs** [comment]
This quite literally defeats the point of dearrow, all thumbnails should be dearrowed so none of them attract your attention more than the other. And if you dont want the videos dearrowed just disable the plugin.

**Comment by surgedevs** [comment]
If you dearrow a video that has already caught your attention with its thumbnail and title you are missing the point of dearrow... dearrow will quite literally not do its intended use case. If you want the original title of a video you can just click the link of the video and it will bring you to the youtube page with the original title...

**Comment by EDM115** [comment]
> dearrow will quite literally not do its intended use case  
  
I get it, but what if i **want** to see the original title and thumbnail  
and then think "okay, but what is it *actually* about ?"  
having dearrow spoiling the content/plot/ending of a video happened too many times for my liking, this is why I consider this an okay-ish use case, where we could bring it on demand when something seems too clickbait but like 90% of the videos my friends send me really don't need this at all

**Comment by electricsteve** [comment]
Yes, I have wanted this since I saw the plugin. Dearrow for me is not how @surgedevs described it, I want to use it like: ok, this video seems high quality, but the title and thumbnail suggest there is something else, I want to know what it is, I click dearrow and I see what it is.

**Comment by EDM115** [comment]
this, crazy how you put it in way simpler words than me 😭🙏

**Comment by electricsteve** [comment]
Was complicated message for me because English isn't my main language.

**Comment by EDM115** [comment]
hmm it seems that if Dearrow is disabled by default, only the derrowed thumbnail appears instead of switching between the OG and the dearrowed one

**Comment by Nuckyz** [comment]
I made a typo haha, will fix
